guidelines/enhancement_template.md

```diff
@@ -72,17 +72,33 @@ this proposal. Describe why the change is important and the benefits to users.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is
-implemented. In each story, explain who the actor is based on their
-role, explain what they want to do with the system, and explain the
-underlying goal they have. Use the standard three part formula:
+Detail the things that people will be able to do if this is implemented and
+what goal that allows them to achieve .In each story, explain who the actor
+is based on their role, explain what they want to do with the system,
+and explain the underlying goal they have, what it is they are going to
+achieve with this new feature.
 
-"As a <role>, I want to <take some action> so that I can <accomplish a
-goal>."
+Use the standard three part formula:
+
+"As a \<role\>, I want to \<take some action\> so that I can \<accomplish a
+goal\>."
 
 Make the change feel real for users, without getting bogged down in
 implementation details.
 
+Here are some example user stories to show what they might look like:
+
+* As an OpenShift engineer, I want to write an enhancement, so that I can share my feature idea
+and get feedback from others.
+* As a product manager, I want the engineers to implement this RFE, so that my customer remains happy.
+* As an administrator, I want a one-click OpenShift installer, so that I can easily set up a new cluster
+without having to follow a long set of operations.
+
+In each example, the persona's goal is clear.
+The engineer wants feedback on their enhancement from their peers.
+The product manager wants to keep their customers happy.
```

Comment:
```suggestion
The product manager wants to keep their customers happy.
```
```suggestion
The product manager wants to keep their customers happy, and providing the RFE they asked for will make them happy.
```



i still think this example is a bit weak, relative to the others, but proposing the edit for consistency.